Explorer/Assets/Scripts/SceneRuntime/Apis/Modules/EngineApiWrapper.cs

```diff
@@ -91,6 +96,17 @@ public ScriptableByteArray CrdtGetState()
             }
         }
 
+        [PublicAPI("Used by StreamingAssets/Js/Modules/EthereumController.js")]
+        public object SendEthereumMessageAsync(int id, string method, string jsonParams)
```

Comment:
It should be moved to another file. E.g. apart from `EngineAPI` we have `RuntimeWrapper` & `RuntimeImplementation`